HARK/models/consumer.py

```diff
@@ -52,7 +52,7 @@
         "dynamics": {
             "b": lambda k, R, PermGroFac: k * R / PermGroFac,
             "m": lambda b, theta: b + theta,
-            "c": Control(["m"]),
+            "c": Control(["m"], upper_bound=lambda m: m),
```

Comment:
@mnwhite @alanlujan91 I wonder what you think about this way of introducing upper/lower bound information on control variables.

Reply:
I think this looks good. 

If we wanted to differentiate between a fixed (real number) upper bound, and a functional upper bound, we could use the term `upper_envelope` for a function/lambda.